HW_2_1/task_3.py

```diff
@@ -0,0 +1,45 @@
+'''Напишите программу для генерации DataFrame с информацией о 35 сотрудниках компании. Фамилии, должности и отделы должны выбираться
+случайным образом из заранее данного списка. Зарплата генерируется как случайное число с нормальным распределением. Необходимо 
+определить минимальну, максимальную и медианную зарплату. Вывести данные о сотрудниках с найденными зарплатами'''
+
+import pandas as pd
+import numpy as np
+import random
+
+len_surname = ['Шостакович', 'Прокофьев', 'Стравинский','Рахманинов',
+             'Скрябин','Римский-Корсаков','Мусоргский','Бородин','Глинка',
+             'Чайковский','Свиридов','Щедрин','Балакиров',
+             'Айвазовский', 'Репин','Шишкин','Серов','Суриков',
+             'Кандинский','Малевич','Кабаков','Васнецов','Врубель','Ге',
+             'Гончарова','Грек','Кончаловский','Малявин',
+             'Иванов','Сидоров','Петров','Смирнов','Кузнецов','Попов','Васильев']
+
+len_department = ['Продажи', 'Мартинг', 'Дизайна', 'Производство', 'Управление сетями']
+
+len_position = ['РукОП', 'РукОМ', 'НачПроизв', 'Директор', 'РукЦП',
+              'Маркетолог', 'Старший маркетолог', 'Младший маркетолог', 'Стажер маркетолог',
+              'Продавец', 'Менеджер по продажам', 'Консультант',
+              'Ведущий дизайнер', 'Дизайнер 1кат', 'Дизайнер 2кат',
+              'Швея', 'Плотник', 'Рукодел', 'Планировщик', 'Маляр 4разряда',
+              'Маляр 3разряда', 'Маляр 2разряда','Маляр 1разряда']
+
+list_zarplata = [np.random.normal(0,1,len(len_surname)) for i in range(len(len_surname))]
+
+
+mean_zarplata = 97750      
+otklonenie = 10000
+
+data_staff = {'Фамилия': [len_surname[i] for i in range(len(len_surname))], 
```

Comment:
Используйте choice.
```suggestion
data_staff = {'Фамилия': np.random.choice(len_position, len(len_surname))], 
```
Ниже так же